examples/basic-wallet/rust-toolchain.toml

```diff
@@ -0,0 +1,5 @@
+[toolchain]
+channel = "nightly-2025-03-20"
+components = ["rustfmt", "rust-src", "clippy"]
+targets = ["wasm32-unknown-unknown", "wasm32-wasip1", "wasm32-wasip2"]
```

Comment:
We probably only need `wasm32-wasip2` mentioned here, no? The other targets would never be used, but would take time to install.